gen_dataset.sh

```diff
@@ -1,3 +1,31 @@
 #!/bin/sh
 
-# Use repo_list and author_list under the config dir
\ No newline at end of file
+cleanup() {
+  rm -rf "$all_repos_temp_dir"
+  rm -rf score-and-rank-contributors
+}
+
+trap 'cleanup' INT TERM EXIT
+
+# Define remote repository for score-and-rank-contributors tool
+TOOL_REPO="https://github.com/anminliu/score-and-rank-contributors.git"
```

Comment:
In addition, I suggest using a specific commit, instead of the head. This shuts down the odds of using a untested version of the tool.